src/views/components/table/Table.web.js

```diff
@@ -489,22 +458,25 @@ class TableView extends Component {
                   disabled={currentPage <= 0}
                 >
                   <Box
-                    backgroundColor={currentPage <= 0 ? '#ddd' : '#5173c6'}
+                    backgroundColor={currentPage <= 0 ? '#ddd' : '#d16360'}
                     padding={10}
                   >
                     <Text
-                      text="Previous"
-                      color="white"
+                      text="<"
```

Comment:
Use an Icon instead of text here.

Reply:
@conorpirie Calvin says "No need icons for previous or next buttons, just << or >> text is sufficient"